examples/with-next-auth/.env.local.example

```diff
@@ -0,0 +1,12 @@
+SITE=http://localhost:3000
```

Comment:
Let's move this one to `.env.development`, as it's not a secret and it will always be this value, and only for development.